.github/workflows/unit-test-workflow.yaml

```diff
@@ -0,0 +1,31 @@
+name: Code Test
```

Comment:
This is a duplicate to what dell/common-github-actions/.github/workflows/go-common.yml@main is running, can this file be removed?

Reply:
I added this file to put in the exclusions on line 31 since this workflow UT check is a new failure on those directories " exclude-directory: "service/mock|provider|core|service/interceptor|common/constants|csi-utils|test/integration|common/k8sutils|core/semver|common/utils|v2". Is there another way to do this other than adding this file?

Reply:
Yes, any exclusions have to be added by a repo admin in the repo settings now. When UT coverage is increased, will any of these packages still be testable?

Reply:
Honestly I don't know as of yet. It's possible some of them are testable since they have go code in them. seem logical to me.